packages/lib/src/components/BacsDD/BacsDD.test.ts

```diff
@@ -0,0 +1,190 @@
+import BacsDD from './BacsDD';
+import { render, screen, within } from '@testing-library/preact';
+import userEvent from '@testing-library/user-event';
+
+describe('Bacs Direct Debit', () => {
+    test('should make a Bacs Direct Debit payment', async () => {
+        const user = userEvent.setup();
+        const onSubmitMock = jest.fn();
+
+        const bacs = new BacsDD(global.core, {
+            modules: { analytics: global.analytics, resources: global.resources },
+            i18n: global.i18n,
+            onSubmit: onSubmitMock,
+            loadingContext: 'https://checkoutshopper-live.adyen.com/checkoutshopper/'
```

Comment:
It needed to be specified for Boleto as far as I remember, since it was doing HTTP requests to get the datasets.
For Bacs might not be the case to pass the live loading URL, but as far as I remember there are warnings in place when this property is omitted.